### ISSUE TITLE
Add ability to split config.rb into multiple files

### ISSUE THREAD

**Issue by westonganger** [algolia/frontman#26]
The config.rb file can get large and unruly. I would like to have the ability to split it into multiple files.

Currently I use the following approach:

```ruby
# config.rb

eval File.read('config/assets.rb')
eval File.read('config/layouts.rb')
eval File.read('config/dynamic_pages.rb')
```

If this approach isnt too dirty maybe we could wrap this up in a little helper method:

```ruby
module Frontman
  class App
    def import_config(path)
      eval File.read(path)
    end
  end
end

# config.rb
import_config "config/assets.rb"
```

**Comment by DevinCodes** [comment]
Hi @westonganger,

Thank you for your suggestion. 😄  I'm curious to see what other people think about adding this.

Personally, I think the approach you currently take is already pretty clean, and I'm not sure a wrapper in Frontman is going to make the code significantly easier to understand or easier to maintain.

However, I do see why there would be a need to split up your config file, and I do think this should be documented. I'm going to add this to the documentation today!

**Comment by DevinCodes** [comment]
Added the documentation for this: https://github.com/algolia/frontman/wiki/Multiple-config-files

**Comment by westonganger** [comment]
I like how you used `run` instead of `eval`.

That being said, the syntax using `File.read` is definitely not intuitive. I think we should improve things for the end user / developer.

**Comment by DevinCodes** [comment]
Feel free to open a PR which adds such a helper method, I would love to see it in action 🙂

**Comment by DevinCodes** [comment]
Closed in #34
